app/helpers/model_helpers.rb

```diff
@@ -0,0 +1,17 @@
+module ModelHelpers
```

Comment:
Thank you @case-eee This explanation helps! It was weird why I had to `include` it manually at multiple places...
FYi @akintner